### PR TITLE
chore add a fallback if item.active does not exist

### PR DESCRIPTION
**Pull request by rabi-siddique**
While testing with the local chain, I encountered this error on the Liquidations Page:
image

During my inspection, I observed that some of the objects used for aggregating data for the graph might be missing certain properties.

### CODE REVIEW

**Review by frazarshad — COMMENTED** [review]
do we need this PR? the issue most likely occurred because you dont have this data indexed on local

**Comment by rabi-siddique** [comment]
> do we need this PR? the issue most likely occurred because you dont have this data indexed on local

Yes, we should consider it. Possible that it can happen in production.

File: src/pages/Liquidated.tsx [hunk 1]
```diff
@@ -64,11 +64,14 @@ export function Liquidated() {
 
   const graphDataList = populateMissingDays(graphDataMap, GRAPH_DAYS);
 
-  const summedGraphDataList = graphDataList.map((item) => ({
-    ...item,
-    active: sum(Object.values(item.active)),
-    liquidated: sum(Object.values(item.active)),
-  }));
+  const summedGraphDataList = graphDataList.map((item) => {
+    const total = item.active ? sum(Object.values(item.active)) : 0;
```
Comment (frazarshad) [inline src/pages/Liquidated.tsx:68]: change default to Nan or undefined. we dont want to falsely indicate 0 as a possible value

Comment (rabi-siddique) [reply src/pages/Liquidated.tsx:68]: It's graph data. 0 is a valid fallback. The graph shows nothing. With NaN it's the opposite.

**Review by frazarshad — APPROVED** [review]

**Pull request merged**